src/environment_setup/conda.mli

```diff
@@ -1,13 +1,20 @@
 open Biokepi_run_environment
 
+type python_version_type = [
+  | `Python2
+  | `Python3
+  | `PythonCustom of string
+  | `PythonToolDependency of string
+]
+
```

Comment:
regular variant it is then. I will give revisit this whole branch once all the implementation is done and will make sure I document all the new parts thoroughly.

Switching between language contexts has this weird effect of involuntarily mixing/matching coding conventions on me. Good catch - will also address current styling issues once all is done.